packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -181,14 +207,26 @@ export const FeedContainer = ({
           )}
           {inlineHeader && header}
           {isV1Search && (
-            <span className="flex flex-row gap-3">
+            <ConditionalWrapper
+              condition={!shouldUseFeedLayoutV1}
```

Comment:
Since we have this edge-case with conditional wrapper we need to test what happens if someone is not on v1 then suddenly get's added.
What I remember the conditional wrapper might omit that evaluation.

Reply:
They need to get refreshed right? Since experiment values are part of the boot, they need to be re-triggered.

Reply:
Yeah think this case will be alright, just not sure on edge-cases.
But maybe we'll see on testing mode.

Reply:
Awesome. Thank you for the insight with regards to the known issue.